frontends/pytorch/csrc/builder/module_builder.cpp

```diff
@@ -95,6 +95,28 @@ ModuleBuilder::startCaptureFunction(std::string &name,
   return std::make_shared<AcapController>(typeMapper, std::move(funcBuilder));
 }
 
+void ModuleBuilder::importFunction(torch::jit::StrongFunctionPtr function) {
+  auto inserter = createInserter();
+  GraphImporter::MlirMappingOptions mappingOptions{
+      context,
+      "", // genericFuncName (default to auto)
+      "", // funcName (default to auto)
+      typeMapper, inserter,
+  };
+  auto graphImporter = GraphImporter::forPythonJitFunc(
+      function.function_, std::move(mappingOptions));
+  graphImporter->initialize();
+  graphImporter->importGenericFunc();
+}
+
+FuncBuilder::Inserter ModuleBuilder::createInserter() {
+  MlirBlock block = getBodyBlock();
+  MlirOperation terminator = this->terminator;
+  return [=](MlirOperation funcOp) {
+    mlirBlockInsertOwnedOperationBefore(block, terminator, funcOp);
```

Comment:
if the name is funcOp, should you assert that it is in fact a function? or just call it `op`.

Reply:
Ah, thanks. Extracted that from elsewhere and copy-pastad.